Setup/creating_test_namespace/runtest.sh

```diff
@@ -46,7 +46,7 @@ rlJournalStart
         rlRun "ocpopDumpOpenShiftClientStatus" 0 "Checking OpenshiftClient installation"
         rlRun "operator-sdk version > /dev/null" 0 "Checking operator-sdk installation"
         rlRun "ocpopCheckClusterStatus" 0 "Checking cluster status"
-        if [ -n "${KONFLUX}" ] && [ -n "${TEST_EXTERNAL_CLUSTER_MODE}" ]; then
+        if [ -n "${KONFLUX}" ] && [ -n "${DOWNSTREAM_IMAGE_VERSION}" ]; then
```

Comment:
We can not do it this way, as this is going to break normal execution with TEST_EXTERNAL_CLUSTER_MODE and KONFLUX.

I suggest doing something like:
if [ -n "${KONFLUX} ] && [ -n "${TEST}" ] ...
   /// apply image mirror
elif [ -n "${KONFLUX} ] && [ -n "${DOWNSTREAM_IMAGE_VERSION}" ] ...
   /// apply image mirror